# Custom Uart Baudrates
# uart_baud_type_t - enumeration
uint8 value

#Pre-Defined ENUM Constants
	uint8 uart_baud_4800       = 3
	uint8 uart_baud_9600       = 4
	uint8 uart_baud_19200      = 5
	uint8 uart_baud_38400      = 6
	uint8 uart_baud_57600      = 7
	uint8 uart_baud_115200     = 8
	uint8 uart_baud_230400     = 9
	uint8 uart_baud_307200     = 10
	uint8 uart_baud_460800     = 11
	uint8 uart_baud_921600     = 12
	uint8 uart_baud_1000000    = 13
	uint8 uart_baud_2000000    = 14
	uint8 uart_baud_3000000    = 15
	uint8 uart_baud_4000000    = 16
